Source/JavaScriptCore/dfg/DFGSpeculativeJIT.cpp

```diff
@@ -3638,14 +3651,12 @@ JITCompiler::Jump SpeculativeJIT::jumpForTypedArrayIsDetachedIfOutOfBounds(Node*
         if (node->arrayMode().isInBounds())
             speculationCheck(OutOfBounds, JSValueSource(), nullptr, outOfBounds);
         else {
-            outOfBounds.link(&m_jit);
+            outOfBounds.link(&m_jit); // This includes resizable arraybuffer's out-of-bounds. But here, we exit only when the buffer is detached.
```

Comment:
nit: Would be cleaner with the comment on the line above.

Reply:
Changed.